assets/tasks/common.json

```diff
@@ -19,7 +19,36 @@
             "args": [
                 "${workspaceFolder}/.conf/projectUpdater.ps1",
                 "${workspaceFolder}",
-                "${workspaceFolderBasename}"
+                "${workspaceFolderBasename}",
+                "0"
+            ],
+            "dependsOrder": "sequence",
+            "icon": {
+                "id": "repo-sync",
+                "color": "terminal.ansiCyan"
+            }
+        },
+        {
+            "label": "try-update-template-accepting-all",
+            "detail": "This task has automation steps to update the project\nbased on the template used to create it.",
```

Comment:
This task has automation steps to update the project\nbased on the template used to create it. Be careful, this one will automatically accept all incoming changes from the updated template

Reply:
Shouldn't the \n be removed here also?